source/FreeplayState.hx

```diff
@@ -47,6 +52,11 @@ class FreeplayState extends MusicBeatState
 			}
 		 */
 
+		 #if desktop
+		 // Updating Discord Rich Presence
+		 DiscordClient.changePresence("In the Menus", null);
```

Comment:
Maybe set this to "In the Freeplay menu" for Freeplay and "In the story mod menu" for story mode?